### ISSUE TITLE
copy() function should support movedown argument

### ISSUE THREAD

**Issue by rayx** [ranger/ranger#1627]
If user press `space` to mark a file, cursor moves down to the next item automatically. I think this is a very convenient feature which saves key typing. It would be good to add the same behavior to `yy`, `ya`, and `yr`.

I understand whether the cursor should move down or not is a subjective matter (see https://github.com/ranger/ranger/issues/1430 for example), but my point is that copy() perhaps should support this behavior and then it's up to user to configure it.

The current workaround is to use `chain` command to bind `yy`, etc, to both `copy` and `move down=1`.

**Comment by toonn** [comment]
This is a good suggestion. Though I personally prefer the more generic `chain` approach.

**Comment by rayx** [comment]
I agree it's more flexible to use `chain`. So I think the real issue is about the default configuration in `rc.conf`. I'm not sure how the other users think about it. Or is it just me?

**Comment by toonn** [comment]
I've never considered it because I usually mark whatever I want and *then* yank it into the copy_buffer. This might make sense with `ya` though. It doesn't with `yy` because the second `yy` would overwrite the first.

**Comment by rayx** [comment]
> I've never considered it because I usually mark whatever I want and _then_ yank it into the copy_buffer. 

I didn't realize it's possible to use it that way :) Unlike long time user like you, who I guess has got accustomed to using marked files for whatever operation, I consider copy buffer as first class citizen. So it's intuitive for me to yank each file to copy buffer rather than mark them first.

**Comment by toonn** [comment]
That's not a bad workflow, the copy_buffer works across directories, which is what many people seem to expect from marking.
